src/main/java/com/snowplowanalytics/snowplow/tracker/Snowplow.java

```diff
@@ -0,0 +1,210 @@
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.tracker;
+
+import com.snowplowanalytics.snowplow.tracker.configuration.EmitterConfiguration;
+import com.snowplowanalytics.snowplow.tracker.configuration.NetworkConfiguration;
+import com.snowplowanalytics.snowplow.tracker.configuration.SubjectConfiguration;
+import com.snowplowanalytics.snowplow.tracker.configuration.TrackerConfiguration;
+import com.snowplowanalytics.snowplow.tracker.emitter.BatchEmitter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Snowplow {
+
+    private static final Map<String, Tracker> trackers = new HashMap<>();
+    private static Tracker defaultTracker;
+
+    /**
+     * @return the stored Trackers (the key is the tracker namespace)
+     */
+    public static Map<String, Tracker> getTrackers() {
+        return trackers;
+    }
+
+    /**
+     * @return the default Tracker, or null if no default is set
+     */
+    public static Tracker getDefaultTracker() {
+        return defaultTracker;
+    }
+
+    /**
+     * Set a specific Tracker instance as the default tracker. The Tracker will be added to the Snowplow class if its
+     * namespace is not already stored.
+     *
+     * The first Tracker created using createTracker() or registered with registerTracker()
+     * will automatically be the default tracker; this method is intended for use
+     * with multiple trackers.
+     *
+     * @param tracker the tracker to use as default
+     */
+    public static void setDefaultTracker(Tracker tracker) {
+        if (!trackers.containsKey(tracker.getNamespace())) {
+            Snowplow.registerTracker(tracker);
+        }
+
+        defaultTracker = tracker;
+    }
+
+    /**
+     * Set a registered Tracker as the default tracker, using its namespace.
+     *
+     * @param namespace the namespace of the tracker to set as default
+     * @return true if the tracker was found and set
+     */
+    public static boolean setDefaultTracker(String namespace) {
+        if (trackers.containsKey(namespace)) {
+            defaultTracker = trackers.get(namespace);
+            return true;
+        }
+
+        return false;
+    }
+
+    /**
+     * Create a Snowplow tracker with default configuration by providing three parameters.
+     *
+     * @param collectorUrl collector endpoint
+     * @param namespace unique identifier for the Tracker instance
+     * @param appId application ID
+     * @return the created Tracker
+     */
+    public static Tracker createTracker(String collectorUrl, String namespace, String appId) {
```

Comment:
This is again a nitpick, but the Android and C++ trackers have the namespace as the first argument in this call (it makes sense to me as first you give the tracker name, then you configure it). Not sure where to put the appId as that is not required in the other trackers, but I think it would be more consistent like this:

```suggestion
    public static Tracker createTracker(String namespace, String collectorUrl, String appId) {
```